e2e/data/expected_context.yaml

```diff
@@ -50,7 +50,7 @@ segmentItems:
       peer: consumer:8082
       skipAnalysis: false
       spanId: 1
-      spanLayer: Http
+      spanLayer: Unknown
```

Comment:
From http to unknown? Why?

Reply:
Just change the default spanlayer of ExitSpan to Unknown, and users can set what they want through setLayer.

Reply:
I see.